umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-umh.md

```diff
@@ -168,17 +244,47 @@ To restore the Node-RED flows, execute the `.\restore-nodered.ps1` script with
 the following parameters:
 
    ````powershell
-   .\restore-nodered.ps1 -KubeconfigPath <PATH_TO_KUBECONFIG> -BackupPath <PATH_TO_BACKUP_FOLDER>
+   .\restore-nodered.ps1 -KubeconfigPath .\k3s.yaml -BackupPath <PATH_TO_BACKUP_FOLDER>
    ````
 
 ### Restore the database
 
-To restore the database, execute the `.\restore-timescale.ps1` script with the
+To restore the `factoryinsight` or `umh_v2` database, execute the `.\restore-timescale.ps1` or  `.\restore-timescale-v2.ps1` script with the
 following parameters:
 
-   ````powershell
-   .\restore-timescale.ps1 -Ip <IP_OF_THE_SERVER> -BackupPath <PATH_TO_BACKUP_FOLDER> -PatroniSuperUserPassword <DATABASE_PASSWORD>
-   ````
+{{< tabs name="restore_db" >}}
+  {{< tab name="factoryinsight" codelang="powershell" >}}
+  .\restore-timescale.ps1 -Ip <IP_OF_THE_SERVER> -BackupPath <PATH_TO_BACKUP_FOLDER> -PatroniSuperUserPassword <DATABASE_PASSWORD>
+  {{< /tab >}}
+  {{< tab name="umh_v2" codelang="powershell" >}}
+  .\restore-timescale-v2.ps1 -Ip <IP_OF_THE_SERVER> -BackupPath <PATH_TO_BACKUP_FOLDER> -PatroniSuperUserPassword <DATABASE_PASSWORD>
+  {{< /tab >}}
+{{< /tabs >}}
+
+You can check the database password by running the following command in your instance's shell:
+```bash
+sudo $(which kubectl) get secret united-manufacturing-hub-credentials --kubeconfig /etc/rancher/k3s/k3s.yaml -n united-manufacturing-hub -o jsonpath="{.data.PATRONI_SUPERUSER_PASSWORD}" | base64 --decode; echo
+```
```

Comment:
I would refactor these steps to first get the password, then run the script. Also, in this case instead of the tabs you can just put the two commands together like this

```powershell
.\restore-timescale.ps1 ...
.\restore-timescale-v2.ps1 ...
```